app-cryptography/sequoia-sq/autobuild/defines

```diff
@@ -4,11 +4,8 @@ PKGDEP="glibc capnproto"
 BUILDDEP="rustc llvm"
 PKGSEC="admin"
 
-CARGO_AFTER="--features crypto-nettle"
+CARGO_AFTER="--features default"
```

Comment:
Why? `crypto-nettle` is still available as the sole default and may change in the future.

I would just leave this change out.